psl-clj/src/psl_clj/core.clj

```diff
@@ -0,0 +1,347 @@
+(ns psl-clj.core
+  "A clojure user interface for PSL"
+  (:require
+   [clj-util.core :as u]
```

Comment:
I am no Clojure specialist so I will yield style issues to you, but I am typically against one letter names (even if it is an alias for a core lib).

Reply:
Done